packages/terra-date-picker/package.json

```diff
@@ -24,8 +24,6 @@
     "react-intl": "^2.3.0",
     "react-redux": "^5.0.4",
     "redux": "^3.6.0",
-    "terra-app-delegate": "^1.9.0",
```

Comment:
Curious, why are these moving to be peerDependencies?

Reply:
No idea. I think I was moving stuff around when I was having problems with hot reloading. 

Updated: https://github.com/cerner/terra-core/pull/1173/commits/0e9b0f65bb69c708fa9c6085f050ffd0bc137b6b